libbz2-rs-sys/src/bzlib.rs

```diff
@@ -579,7 +579,6 @@ pub(crate) struct SaveArea {
     pub es: i32,
     pub N: i32,
     pub curr: i32,
-    pub zt: i32,
```

Comment:
:+1: also unused in the original C code.

Reply:
yeah, I'm really not sure what is happening there.

Reply:
as in, why would they leave that field around if it's unused

Reply:
My guess would be that it was used in the past and the compiler didn't give a warning when it became unused due to the store/restore dance.